packages/lending/src/adapter.config.ts

```diff
@@ -39,13 +48,24 @@ export const configs: Config[] = [
     secondaryStablecoin: avalancheTokens['DAI.e'],
     primaryNonstablecoin: avalancheTokens['WETH.e'],
   },
-
   {
     chainId: common.ChainId.metis,
     primaryStablecoin: metisTokens['m.USDC'],
     secondaryStablecoin: metisTokens['m.DAI'],
     primaryNonstablecoin: metisTokens.WETH,
   },
+  {
+    chainId: common.ChainId.base,
+    primaryStablecoin: baseTokens.DAI,
+    secondaryStablecoin: baseTokens.USDC,
+    primaryNonstablecoin: baseTokens.WETH,
+  },
+  {
+    chainId: common.ChainId.gnosis,
+    primaryStablecoin: gnosisTokens.USDC,
+    secondaryStablecoin: gnosisTokens.USDT,
+    primaryNonstablecoin: gnosisTokens.WETH,
+  },
```

Comment:
Seems `WXDAI` is most commonly stablecoins on Gnosis chain, do we want to use it?

Reply:
fixed